common/core/gpudevice.cpp

```diff
@@ -57,13 +69,15 @@ bool GpuDevice::Initialize() {
     display_manager_->RemoveUnreservedPlanes();
   }
 
-  lock_fd_ = open("/vendor/hwc.lock", O_RDONLY);
+  // Ignore all updates by default
+  ResetAllDisplayCommit(false);
+  lock_fd_ = open(HWC_LOCK_FILE, O_RDONLY);
```

Comment:
It seems a defect to me that if no lock_fd_ HWC resist to initial
If no lock_fd, HWC should open gpu_fd and become master I think
No hwc.lock should mean the system want no other application to hold drm master

Reply:
Get it. If  lock_fd_ == -1, HWC will take drmmaster, and forcerefresh